package.json

```diff
@@ -29,10 +29,10 @@
     "jstransform": "./scripts/jstransform.sh",
     "jstransform-watch": "./scripts/jstransform.sh --watch",
     "browserify-all": "npm run browserify && npm run browserify-test",
-    "browserify": "browserify -r .:pileup -v -t [ envify --NODE_ENV production ] --debug -o dist/pileup.js",
+    "browserify": "browserify -r .:pileup -v -g [ envify --NODE_ENV production ] -g uglifyify --debug -o dist/pileup.js",
```

Comment:
does this lead to double uglification (first via uglifyify and then directly via uglify)?